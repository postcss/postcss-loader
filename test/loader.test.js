'use strict'

const webpack = require('./helpers/compiler')
const { loader } = require('./helpers/compilation')

describe('Loader', () => {
  test('Default', () => {
    const _ = /style\.(exec\.js|js)$/
    const config = {
      loader: {
        options: {
          plugins: []
        }
      }
    }

    return webpack('css/index.js', config).then((stats) => {
        const src = loader(stats).src

        expect(src).toEqual("module.exports = \"a { color: black }\\n\"")
        expect(src).toMatchSnapshot()
      })
  })
})
