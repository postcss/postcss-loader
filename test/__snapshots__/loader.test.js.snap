// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Default 1`] = `"module.exports = \\"a { color: black }\\\\n\\""`;
